build.sbt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ThisBuild / name         := "fa-db"
+ThisBuild / organization := "za.co.absa"
+
+lazy val scala211 = "2.11.12"
+lazy val scala212 = "2.12.12"
+
+ThisBuild / scalaVersion := scala211
+ThisBuild / crossScalaVersions := Seq(scala211, scala212)
+ThisBuild / publish := {}
```

Comment:
I think this line should be removed

Reply:
I guess. I copped other projects mostly. 😄

Reply:
IIRC this was made so it would not publish a root project. But as this project is just a root project without project definition, I agree with Adrian.

Then again cleaning of SBT can come later.

Reply:
SBT fixing do need to come. Planned as the third and last PR before first release.
Among other to make it a multi-package project - Core and Slick